vsintegration/src/FSharp.Editor/LanguageService/FSharpProjectOptionsManager.fs

```diff
@@ -17,6 +18,213 @@ open Microsoft.VisualStudio.FSharp.Editor.SiteProvider
 open Microsoft.VisualStudio.LanguageServices
 open Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
 open Microsoft.VisualStudio.Shell
+open System.Threading
+open Microsoft.VisualStudio.Shell.Interop
+open Microsoft.VisualStudio.LanguageServices.Implementation.TaskList
+
+[<AutoOpen>]
+module private FSharpProjectOptionsHelpers =
+
+    let mapCpsProjectToSite(workspace:VisualStudioWorkspaceImpl, project:Project, serviceProvider:System.IServiceProvider, cpsCommandLineOptions: IDictionary<ProjectId, string[] * string[]>) =
```

Comment:
This looks like a refactor of, do we still use the original? 

https://github.com/Microsoft/visualfsharp/blob/81694900dcec17a169f390dafcb8a1a8a7d08b31/vsintegration/src/FSharp.Editor/LanguageService/ProjectSitesAndFiles.fs#L136

Reply:
We don't use the original. But in my second PR: https://github.com/Microsoft/visualfsharp/pull/5845 - ProjectSitesAndFiles.fs get completely removed.